packages/vanilla-components/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@saasquatch/vanilla-components",
   "title": "Vanilla Components",
-  "version": "1.2.8",
+  "version": "1.2.9-4",
```

Comment:
revert these changes - vanilla-components are updated in a different branch for feature enforcement